tests/test_auth.py

```diff
@@ -42,6 +42,13 @@
 
 class TestAuthUtils(OsfTestCase):
 
+    def test_generate_csl_given_name(self):
+        given_name = 'Cause'
+        middle_names = 'Awesome'
+        suffix = 'Jr.'
+        csl_given_name = generate_csl_given_name(given_name, middle_names, suffix)
+        assert_equal(csl_given_name, 'Cause A ,Jr.')
```

Comment:
This doesn't look right. Shouldn't it be `'Cause A, Jr.'`?

Also, there should probably be a more comprehensive set of these tests to cover all of the different combos of given, middle, and suffix, since we're here. Maybe ask someone from QA or Product what they would expect for various name combinations to return so we can make sure everyone is happy.